apps/server/src/modules/tool/service/tool-validation.service.spec.ts

```diff
@@ -105,6 +105,26 @@ describe('ToolValidation', () => {
 				await expect(result).resolves.not.toThrow();
 			});
 
+			it('should validate the default with the regular expression', async () => {
```

Comment:
describe('when validating default value with regex', ...)
     describe('default values matches the regex)'
        it('should not throw'...)
     describe('default values not match the regex'...)
        it('should throw...)